bootstrap.inc.sh

```diff
@@ -112,9 +113,37 @@ function _bootstrap_configure_common() {
     _bootstrap_download "_common/$common_file" "$BOOTSTRAP_COMMON/$common_file"
   done
 
+  _bootstrap_echo "Downloading _common/assets/sil-logos-2024/ files"
+
+  _bootstrap_download_directory "_common/assets/sil-logos-2024" "$BOOTSTRAP_COMMON/assets/sil-logos-2024"
+
   _bootstrap_echo "All _common files downloaded"
 }
 
+#
+# Download a directory
+#
+function _bootstrap_download_directory() {
+  local source_dir="$1"
+  local target_dir="$2"
+
+  mkdir -p "$target_dir"
+
+  local ASSET_LOGOS=(
+    sil-logo-abbysinica.png
+    sil-logo-andika-v1.png
+    sil-logo-andika-v2.png
+    sil-logo-annapurna.png
+    sil-logo-tai-heritage-pro.png
+  )
```

Comment:
This is not generic -- it is limited to specific files in the sil-logos-2024 directory, so it doesn't make sense to call this function _bootstrap_download_directory. It would be better to move that variable into the caller

Reply:
I'll look into this.

Reply:
I think addressed in #33